src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -102,6 +102,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
     <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
+
+    <WarningsAsErrors>NU1605</WarningsAsErrors>
```

Comment:
I don't think that not doing this for F# is the right approach. This is all package resolution and it should behave the same across all languages. However, it might be the easier way for now. @KevinRansom, is there a reason why fsc does not behave he same way as csc and vbc and ignores unknown identifiers? Would it be possible to make it do so? Also, @srivatsn, is there another property that we could use if none of the alternatives above work?